### PR TITLE
add Terje to partners + update pilot page for 2020.09 version of pilot repo

### PR DESCRIPTION
**Pull request by terjekv**

### CODE REVIEW

File: docs/pilot.md [hunk 1]
```diff
@@ -60,32 +57,38 @@ The container image can be used directly by Singularity (no prior download requi
...
 If all goes well, you should see output like this:
 
 ```shell
-Singularity> source /cvmfs/pilot.eessi-hpc.org/2020.08/init/bash
-
-Found EESSI pilot repo @ /cvmfs/pilot.eessi-hpc.org/2020.08!
-Derived subdirectory for software layer: x86_64/intel/haswell
-Using x86_64/intel/haswell subdirectory for software layer (HARDCODED)
+Singularity> source /cvmfs/pilot.eessi-hpc.org/2020.09/init/bash
+Found EESSI pilot repo @ /cvmfs/pilot.eessi-hpc.org/2020.09!                                                                                                          [68/881]
```
Comment (boegel) [inline docs/pilot.md:77]: @terjekv What's the `[68/881]` bit?

File: docs/pilot.md [hunk 2]
```diff
@@ -107,78 +110,75 @@ Any feedback is welcome, and questions or problems reports are welcome as well,
...
 ### Available software
 
-*(last update: Sept 3rd 2020)*
+*(last update: Sept 30th 2020)*
 
 ```
-[EESSI pilot 2020.08] $ module avail
+[EESSI pilot 2020.09] $ module avail
 
 -------------------------------------- /cvmfs/pilot.eessi-hpc.org/2020.08/software/x86_64/intel/haswell/modules/all --------------------------------------
```
Comment (boegel) [inline docs/pilot.md:118]: use same arch as above (`cascadelake`)?

File: docs/pilot.md [hunk 3]
```diff
@@ -107,78 +110,75 @@ Any feedback is welcome, and questions or problems reports are welcome as well,
...
-packagepath    (E) = /tmp/easybuild/packages
-prefix         (E) = /tmp/easybuild
-repositorypath (E) = /tmp/easybuild/ebfiles_repo
-robot-paths    (D) = /home/eessi/easybuild-easyconfigs/easybuild/easyconfigs
-rpath          (E) = True
-sourcepath     (E) = /tmp/easybuild/sources
-sysroot        (E) = /cvmfs/pilot.eessi-hpc.org/2020.08/compat/x86_64
-trace          (E) = True
+buildpath         (D) = /tmp/terjekv/singularity-home/.local/easybuild/build
+containerpath     (D) = /tmp/terjekv/singularity-home/.local/easybuild/containers
```
Comment (boegel) [inline docs/pilot.md:170]: Please edit a bit to make it "anonymous" (no `terjekv`, like it was before)

**Review by boegel — CHANGES_REQUESTED** [review]
In the available software section, we should mention that we have installed the following "applications": `GROMACS`, `OpenFOAM` (two versions), `R` 4.0.0, and `Bioconductor` 3.11 (`R-bundle-Bioconductor` module), and clarify that everything else are dependencies.

We should also add a new subsection that lists for which architectures the software was installed + mention the `aarch64/generic` and `x86_64/generic` fallback parts.

Maybe we should also keep the original `2020.08` as `pilot-2020.08.md` so we can link to it, and highlight the differences between `2020.09` and `2020.08` (no more `$LD_LIBRARY_PATH`, `filter-deps`, more archs, more software, etc.)?

File: docs/pilot.md (hunk 1 above)
Comment (terjekv) [reply docs/pilot.md:77]: That's tmux being "helpful". Fixed!

File: docs/pilot.md (hunk 2 above)
Comment (terjekv) [reply docs/pilot.md:118]: Yeah, fixing them all. Sloppy, sorry.

File: docs/pilot.md (hunk 3 above)
Comment (terjekv) [reply docs/pilot.md:170]: Good point! Will do.

**Comment by terjekv** [comment]
Updated, but don't we have git for history?

**Comment by boegel** [comment]
@terjekv I understand your comment about Git history, but it may be useful for people to be able to compare the `2020.09` with `2020.08` version just from the docs, without having to wade through history in the Git repo.

That said, we can add back `pilot-2020.08.md` later, since we have what we need in Git history. :)

Thanks a lot for the effort spent in updating this page! :+1:

**Review by boegel — APPROVED** [review]

**Pull request merged**